feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableListItem.kt

```diff
@@ -55,6 +60,7 @@ fun TimetableListItem(
     onBookmarkClick: (TimetableItem) -> Unit,
     chipContent: @Composable RowScope.() -> Unit,
     modifier: Modifier = Modifier,
+    highlightIndexLange: (String) -> IntRange = { IntRange.EMPTY },
```

Comment:
How about having `searchingText: String? = null` instead of highlightIndexLange? I think it looks easier.

Reply:
I agree with you, I'll do coding again.

Reply:
Instead of `String?`, I choose `SearchListUiState`, which is value class of `String` and has some extension functions.

Reply:
I think you don't have to use `UiState` name for this. You can create `SearchQuery` or something. This is because we don't use UiState for components. And you can write some classes as usual composable classes in the official library.
https://github.com/DroidKaigi/conference-app-2023#component

Reply:
I changed from `highlightIndexLange: (String) -> IntRange = { IntRange.EMPTY },` to `highlightQuery: SearchQuery = SearchQuery.EMPTY,`.
How about it?

https://github.com/S-H-Y-A/conference-app-2023/blob/84b1ebd52b0aab801dccac2d8787e7f3fea85ac1/feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableListItem.kt#L64

Reply:
There is a very little point. We don't use snake case for Compose code. 
https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md#singletons-constants-sealed-class-and-enum-class-values

Reply:
Ops, that's my fault.
I have to read API guidelines again.
Thank you for your advice.

Reply:
I converted it into camel case and pushed.